### ISSUE TITLE
Principal 2: Operable 2.1 Keyboard accessibile

### ISSUE THREAD

**Issue by ncunning** [PureWeen/A11YTools#2]
WCAG description: | All functionality of the content is operable through a keyboard interface without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints.

Note: When testing for mobile devices, it is assumed that the swipe gesture functionality when VoiceOver is turned on presents the same accessible content as a user with a physical keyboard would be able to access using the tab and arrow keys. If a piece of content is not accessible through swipe gestures, then it is assumed it will also not be accessible through a keyboard. In addition to the above, testing has been done with a Bluetooth keyboard connected to an iPhone 7 with the Full Keyboard Access accessibility mode switched on.
-- | --
WCAG recommendations: | All page functionality is available using the keyboard, unless the functionality cannot be accomplished in any known way using a keyboard (e.g., free hand drawing).Page-specified shortcut keys and access keys (access keys should typically be avoided) do not conflict with existing browser and screen reader shortcuts.
